mantidproject/mantid
# LoadAndMerge fix for ignored loader version

**Description of work.**

This PR fixes the issue of LoadAndMerge algorithm ignoring user's choice of the loader version. The issue was passing arguments not by keyword but by their position.

**To test:**

Repeat the test from the issue #32909, this time when LoaderVersion is set to 1, there should be a clear error when an attempt is made to load the suggested file.

Fixes #32909. 

Release note added.



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
